SIT.Manager/Views/Play/CharacterSummaryView.axaml

```diff
@@ -11,9 +11,11 @@
 			Margin="0,8"
 			Background="Black">
 		<Grid ColumnDefinitions="Auto, *, Auto">
-			<TextBlock Grid.Column="0"
-					   VerticalAlignment="Center"
-					   Text="{Binding Profile.Side}"/>
+			<Image Grid.Column="0"
+					VerticalAlignment="Center"
+					Source="{Binding SideImage}"
+					MaxWidth="80"
+					MaxHeight="80"/>
```

Comment:
If you set Stetch="Uniform" then I think that might help the images be clearer

Reply:
I *might* have slightly copied SPT's launcher and it looks mildly better? Idk exactly why their scaling looks smother compared to ours